lib/nine_one_one/configuration.rb

```diff
@@ -50,6 +55,16 @@ def validate_slack_enabled
         raise ConfigurationError, 'Incorrect Slack webhook URL'
       end
     end
+
+    def validate_slack_opts
+      unless slack_username.nil? || slack_username.is_a?(String)
+        raise ConfigurationError, "Illegal 'slack_username' value: #{slack_username}"
+      end
+
+      unless slack_channel.nil? || slack_channel.is_a?(String)
```

Comment:
I don't like `unless` with code block. I'd use `if` here because it's more legible IMO.